### ISSUE TITLE
When my sketch renders a "jr.background()" I always get a “java.lang.Null.Pointer.Exception”.

### ISSUE THREAD

**Issue by jacbouzada** [joonhyublee/joons-renderer#10]
Congratulations for your "joonsrenderer" library. It´s amazing.

I have a question to make you about the jr.background() method.
I´m using Processing 2.2.1 on a MacBook Pro Retina display with OS X 10.10.2. 

When my sketch renders a "jr.background()" I always get a “java.lang.Null.Pointer.Exception”.

In the example “joons_renderer_DOF_example” I get the error that had copy/paste below.
But if I delete the line "jr.background(0)” the sketch renders perfectly !!!! 
I get the same result as you because of the black background.

When I try to render the example “joons_renderer_fill_example“ I get the same error 
BUT if I go to the "void draw" block and I delete the line 
“ jr.background(0, 255, 255);” then the sketch renders perfectly !!!! (but in this case with a black background).

My level of processing is low. Do you know what I am doing wrong?

Thank you for your great library.

java.lang.RuntimeException: java.lang.NullPointerException
	at com.jogamp.common.util.awt.AWTEDTExecutor.invoke(AWTEDTExecutor.java:58)
	at jogamp.opengl.awt.AWTThreadingPlugin.invokeOnOpenGLThread(AWTThreadingPlugin.java:103)
	at jogamp.opengl.ThreadingImpl.invokeOnOpenGLThread(ThreadingImpl.java:206)
	at javax.media.opengl.Threading.invokeOnOpenGLThread(Threading.java:172)
	at javax.media.opengl.Threading.invoke(Threading.java:191)
	at javax.media.opengl.awt.GLCanvas.display(GLCanvas.java:541)
	at processing.opengl.PJOGL.requestDraw(PJOGL.java:688)
	at processing.opengl.PGraphicsOpenGL.requestDraw(PGraphicsOpenGL.java:1651)
	at processing.core.PApplet.run(PApplet.java:2256)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
	at processing.opengl.PJOGL.clearDepth(PJOGL.java:2404)
	at processing.opengl.PGraphicsOpenGL.backgroundImpl(PGraphicsOpenGL.java:5335)
	at processing.core.PGraphics.backgroundFromCalc(PGraphics.java:6983)
	at processing.core.PGraphics.background(PGraphics.java:6929)
	at processing.core.PApplet.background(PApplet.java:15139)
	at joons.JoonsRenderer.background(JoonsRenderer.java:95)
	at joons_renderer_DOF_example.draw(joons_renderer_DOF_example.java:58)
	at processing.core.PApplet.handleDraw(PApplet.java:2386)
	at processing.opengl.PJOGL$PGLListener.display(PJOGL.java:862)
	at jogamp.opengl.GLDrawableHelper.displayImpl(GLDrawableHelper.java:665)
	at jogamp.opengl.GLDrawableHelper.display(GLDrawableHelper.java:649)
	at javax.media.opengl.awt.GLCanvas$10.run(GLCanvas.java:1289)
	at jogamp.opengl.GLDrawableHelper.invokeGLImpl(GLDrawableHelper.java:1119)
	at jogamp.opengl.GLDrawableHelper.invokeGL(GLDrawableHelper.java:994)
	at javax.media.opengl.awt.GLCanvas$11.run(GLCanvas.java:1300)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:302)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:733)
	at java.awt.EventQueue.access$200(EventQueue.java:103)
	at java.awt.EventQueue$3.run(EventQueue.java:694)
	at java.awt.EventQueue$3.run(EventQueue.java:692)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:703)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)

**Comment by joonhyublee** [comment]
Hey man thanks for your interest. I am currently not maintaining the library, but others have noted and fixed a similar issue, since its open source and all you know. I have not updated the download page since, but the source code on the repo has been updated, so you can try downloading the latest version directly from repo. See how that works and tell me how it went. Thanks!

**Comment by jacbouzada** [comment]
I have installed Eclipse and learnt how to deal with it.  For me was a great advanced!!!
After learnt a bit about java classes, inheritance, encapsulation,... I began to study some tutorials about how to prepare libraries for Processing.
And finally.... after a lot of attempts and built.propieties configurations,etc... succesfully run as ant built !!!!
Happy because the library works fine and happy because I learnt a lot about Eclipse.
Thank you!
